src/CommandRunner/Git.php

```diff
@@ -61,11 +61,13 @@ public function getRemoteUrl(string $remote = 'origin'): string
      * Check if the git repository in the provided directory has any uncommitted changes
      *
      * @param string $directory A directory containing a git repository
+     *
      * @return bool True if there are changes, false otherwise
      */
     public function hasUncommittedChanges(string $directory): bool
     {
         $changes = $this->run($directory, "git diff-index --name-only HEAD");
+
```

Comment:
Same here

Reply:
It is also a CS fix to be coherent with the rest of the codebase.
It's a common CS (for instance in Symfony).